wordpress-mobile/WordPress-iOS
# [Gutenberg] Implementing Ghost loading view for Gutenberg

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/1179

Gutenberg-mobile side PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2382

This PR implements a loading view for Gutenberg using the Ghost Views used in the WPiOS App.

To test:
- Run the app.
- Open a post or page with Gutenberg.
- Check that the loading view shows similar to the gif attached.
- Check light and dark mode, portrait and landscape.

![skeleton](https://user-images.githubusercontent.com/9772967/84534492-4816cd00-acea-11ea-95c5-cd88eb69693b.gif)


PR submission checklist:
